nncf/quantization/algorithms/channel_alignment/algorithm.py

```diff
@@ -373,12 +387,13 @@ def get_statistic_points(self, model: TModel, graph: NNCFGraph) -> StatisticPoin
         statistic_container = StatisticPointsContainer()
         for conv_in, add_in, _ in self._get_node_pairs(graph):
             target_point, node_in = self._get_target_point_and_node_in(conv_in, add_in)
+
             channel_axis = conv_in.metatype.output_channel_axis
-            reduction_axes = list(range(len(graph.get_output_edges(node_in)[0].tensor_shape)))
-            reduction_axes.remove(channel_axis)
+            activation_shape = list(range(len(graph.get_output_edges(node_in)[0].tensor_shape)))
+            reduction_shape = self._backend_entity.get_channel_agnostic_reduction_axes([channel_axis], activation_shape)
```

Comment:
I'm guessing that `get_channel_agnostic_reduction_axes` should return `reduction_axes`

Reply:
Done